content-signup/README.md

```diff
@@ -0,0 +1,73 @@
+# Automatically send an email following up with content to people who signup via a form.
+
+For marketing activities, you may want to automate the sending of key links dynamically to content such as videos, whitepaper links, blogs, PDF summaries, etc based on what people mark as their specific interest in your contact-us form linked on your website, email signature, or wherever you share the form's link.
+
+You can also ask if they would like to subscribe to your mailing list for future content.
+
+(last updated June 28 2019)
+
+## Setup
+
+<!-- [START setup] -->
+
+**First**, set up the **Google Form** and response spreadsheet:
+
+1. Open [Google Forms](https://docs.google.com/forms/u/0/) in your browser.
+1. **Create a new form** called **"Sign-up for information form"**
+1. Create the following **4** fields with the respective **title** and **field type**:
+   * **My name** (field type: **short answer**)
+
+   * **Email address** (field type: **short answer**)
+
+   * **Topics I would like to learn about** (field type: **checkboxes** with the following options. Maintain the paranethesis and capital letters as they are, the code is looking for this style).
+      * (NUTRITION) Raw vegan recipes
+      * (REPROGRAMMING HABITS) The Zero Waste white paper
+      * (URBAN FOOD) A community garden how-to
+      * (WATER DESIGN) Grey water plumbing supplies
+
+   * **I would also like to keep receiving information in the future** (field type: **drop down** with the following **2** options...)
+      * Yes add me to the mailing list.
+      * No just this one time send me the assets.
+
+1. Select a **Google Spreadsheet** as the source for **responses**.
+
+1. Return to the Google Form and respond to all 4 fields
+populated with events, ensure to use an email address you have access to. 
+
+
+Next, paste the **Apps Script code in the script editor** (this type of script is called a **'container-bound script'** because it lives in a spreadsheet):
+
+1. From your Google Spreasheet click **Tools > Script Editor** 
+1. Paste the Apps Script source code that is listed in the file **src** in this GitHub.
+
+Then run the script:
+
+1. Click the **Select function** dropdown and select **setup**.
+1. Click the Run button (►) > **SendCampaignEmail**
+1. When prompted, click the **Review permissions** button.
+1. Select your Google account from the list.
+1. Click the **Allow** button.
+
+Setup a trigger to **automate the script's execution** whenver a new form entry is submitted moving forward:
+
+1. Click **Edit > Current Project's Triggers**
+1. Click **Add trigger** (in the corner right bottom)
+1. Select the following defaults:
+   * Choose which function to run > **SendCampaignEmail**
+   * Choose which deployment should run > **Head**
+   * Select event source > **From Spreadsheet**
+   * Select event type > **On form submit**
```

Comment:
If this uses onFormSubmit -- do you need to have an onFormSubmit(e) in your code like in the quickstart here:
https://developers.google.com/apps-script/quickstart/forms